src/Diagrams/RubiksCube/Model.hs

```diff
@@ -29,6 +29,17 @@ module Diagrams.RubiksCube.Model (
   -- ** By position
   , centerFacets, cornerFacets, edgeFacets
   , sideCorners, sideEdges
+  -- ** By rows
+  , upRows
+  , middleRows
+  , downRows
+  -- ** By cols
+  , leftCols
+  , _centerCols
```

Comment:
I believe I used the underscore as prefix to prevent an "unused declaration" compiler warning. Now that this (and `_betweenCols`) are exported (and therefore used) I think we should remove the prefix.

Reply:
Good remark! Commit below.